### ISSUE TITLE
JavaScript example improvements

### ISSUE THREAD

**Issue by splatterxl** [JapiRest/docs.japi.rest#3]
Currently, the JavaScript examples for all the endpoints use [`node-fetch`](https://npm.im/node-fetch), which
1. needs to be downloaded from an external source, and
1. some users may have a personal or company preference for another HTTP library.

I think the examples should either use the builtin `http`/`https` packages or just the pre-defined Web Fetch API, as expecting users to be using node-fetch may mislead beginner programmers into thinking that Node.js is the only way to use the API, or `node-fetch` is the only HTTP package compatible with the API, or some other bullsh*t.

**TL;DR:** using `node-fetch` for the examples is suboptimal, use a builtin function instead. I'd recommend just removing the `const fetch = ...` line.

*Whatever you do, **do not use XMLHTTPRequest** in the examples (shudder)*

**Comment by DanPlayz0** [comment]
Fixed with commit `c536b28d957a13acb8a22a03cefc1760e881ac2d`

**Comment by nearlySplat** [comment]
(You can add `closes #pr_id` to commits to close them automatically btw)
